src/main/resources/synthea.properties

```diff
@@ -27,7 +27,9 @@ exporter.practitioner.fhir_stu3.export = false
 exporter.practitioner.fhir_dstu2.export = false
 exporter.encoding = UTF-8
 exporter.csv.export = false
-# if exporter.csv.append_mode = true, then each run will add new data to any existing CSVs. if false, each run will clear out the files and start fresh
+exporter.json.export = false
```

Comment:
This is super pedantic but can these new properties be moved up or down so they don't split the CSV-specific properties?